frontend/src/components/common/Resource/ResourceTable.tsx

```diff
@@ -86,7 +86,11 @@ function Table(props: ResourceTableProps) {
       case 'namespace':
         return {
           label: t('glossary|Namespace'),
-          getter: (resource: KubeObject) => resource.getNamespace(),
+          getter: (resource: KubeObject) => (
+            <Link routeName={`namespace`} params={{ name: resource.getNamespace() }}>
```

Comment:
Nit: You don't need the string literal for the routeName. `routeName="namespace"` should do it.